packages/api/src/controllers/asset.ts

```diff
@@ -343,7 +358,7 @@ app.get("/:id", authorizer({}), async (req, res) => {
     );
   }
 
-  res.json(asset);
+  res.json(db.asset.cleanWriteOnlyResponse(asset));
```

Comment:
This is not necessary as the `mung` JSON middleware already does that.

https://github.com/livepeer/livepeer-com/blob/98652ad082a738f3697e17862192994946786184/packages/api/src/controllers/asset.ts#L212